pipeline/scripts/ci/getPipelineStages.py

```diff
@@ -71,6 +71,17 @@ def fetch_stages(args):
     overrides = json.loads(args.get("overrides", {}))
     overrides_log = json.loads(args.get("overrides", {}))
     cloud_type = "openstack"
+
+    # fetch overrides provided in metadata file
+    metadata_overrides = list(filter(lambda d: "overrides" in d, data))
```

Comment:
The initial idea of the metadata file was to hold a list of suites. As we are expanding, IMO we should have something like this in our file
```
overrides:
  custom-config: "promtail_image=brew.registry.redhat.io/rh-osbs/promtail:ceph-6-rhel-9"
  log-level: 
suites:
  - suite1
  - suite 2
```

This would allow us to have `overrides` as a key and the existing list under `test-suites. This would change other data files. However, it gives us the scope of expansion in the future.

Probably it enhances the readability of the file.

Reply:
Done. please let me know if the current format looks okay.